### DISCUSSION TITLE
Pop-up vs Open

### DISCUSSION THREAD

**Discussion by Solidwire99** [libre-tube/LibreTube#2344] [Q&A] [answered]
I was using Open to Send the video to an app called Web Video Caster to watch videos on the big screen... 
The icon now reads Pop-up & simply seems go to Picture- in - Picture which was easily accomplished previously by minimizing & didn't require a button.

Must I downgrade the version or is there another method to sending the video to a different app?

Thank you for your time and consideration.
Don Jude
SoLidWire

**Comment by Bnyro** [comment, marked as answer]
There's currently no way to do so, maybe it will be readded to a less important position / smaller button somewhere, see #2292.
